sys-kernel/coreos-kernel/coreos-kernel-5.8.14-r1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
-COREOS_SOURCE_REVISION=""
+COREOS_SOURCE_REVISION="-r1"
```

Comment:
I think it's ok to leave the package version at the same revision that it doesn't conflict with https://github.com/flatcar-linux/coreos-overlay/pull/647

Reply:
Removed the revision changes.

Reply:
It anyway conflicts, no idea why, but at least we tried :smile:

Reply:
Interesting. Manual rebase work just well.
I simply pushed a new rebase to https://github.com/flatcar-linux/coreos-overlay/pull/647